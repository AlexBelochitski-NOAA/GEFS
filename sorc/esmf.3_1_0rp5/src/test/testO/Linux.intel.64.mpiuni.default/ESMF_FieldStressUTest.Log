20190610 112239.501023 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112239.501286 INFO     PET0  Beginning Test, file ESMF_FieldStressUTest.F90, line          51
20190610 112239.501304 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112239.501322 INFO     PET0  Number of failed tests:           0
20190610 112239.501328 INFO     PET0  Ending Test, file ESMF_FieldStressUTest.F90, line          87
