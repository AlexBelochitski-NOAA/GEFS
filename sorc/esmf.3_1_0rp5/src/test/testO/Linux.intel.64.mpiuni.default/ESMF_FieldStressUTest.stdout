  Beginning Test, file ESMF_FieldStressUTest.F90, line          51
  NUMBER_OF_PROCESSORS           1
  Number of failed tests:           0
  Ending Test, file ESMF_FieldStressUTest.F90, line          87
